src/getExtension/getExtension.ts

```diff
@@ -5,7 +5,7 @@ import {
   PubSubSessionV2,
 } from '../types/types'
 
-const apiWindow = window as Window & ApiWindow
+const apiWindow = (global.window ?? {}) as Window & ApiWindow
```

Comment:
I feel that the window being potentially absent is some TS misconfiguration

Reply:
yeah the fact that we have code for backende and for frontends in the same package does not make things easier. I had the issue here that (because it's compiled to cjs) importing the library would crash even if I did not use anything from this file

Reply:
ideas I had to fix this include:
- subpath exports instead of an index file for everything 
- esm
- monorepo (yeeeey!)

Reply:
my issue is less with the types not being applied, but more with the window object actually not existing when we load this from node.js, which was breaking the code

Reply:
because it's all exported via the same index file, which, as I said, probably requires one of the three possible solutions I suggested to be fixed

Reply:
All right, then just use `globalThis` instead of `window`. For us on the frontend, `window` is just an alias for the missing global object.

Reply:
ah good to know!